Packs/CortexXDR/ReleaseNotes/6_1_7.md

```diff
@@ -0,0 +1,12 @@
+
+#### Playbooks
+
+##### New: Cortex XDR IOCs - Disable expired IOCs in XDR
+
+- New: This playbook will disables indicators in XDR after they have expired from XSOAR. (Available from Cortex XSOAR 6.10.0).
```

Comment:
you will need to create the release notes again with the new descriptions

Reply:
Done.